.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -136,7 +136,7 @@ const getPipeline = (filename: string, removeSteps = true) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/ux_plugin_e2e.yml'));
     }
 
-    if (await doAnyChangesMatch([/^x-pack\/plugins\/observability_solution/])) {
+    if (await doAnyChangesMatch([/^x-pack\/plugins\/observability_solution/, /^package.json/])) {
```

Comment:
We probably want to monitor `yarn.lock` as well. We can update the lock file without touching `package.json`

Reply:
done